### PR TITLE
Adds shimmering orbs.

### PR DESCRIPTION
**Pull request by ShibberinoShipperino**
🆑
Adds shimmering orbs, small floating orbs filled with bioluminescent growths.
🆑

Smalls rocks that could be used as a light source, most likely to be found on asteroids.
They have not been mapped on any away missions as of yet.
Made this sprite a while back and never got to use it.

Images: 
ShimmeringOrb
Capture

### CODE REVIEW

**Comment by Glloyd** [comment]
Requested changes will need to be made before this is merged

**Comment by ShibberinoShipperino** [comment]
Sorry, I've been rather busy with University Graduation stuff. All changes complete, moved to Urist's folders. I'll make sure to keep to that in future for ease.

**Comment by Glloyd** [comment]
No worries, there's no time limit on PRs here.

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -365,4 +365,21 @@
...
+	icon_state = "shimmering_orb"
+	plane = ABOVE_HUMAN_PLANE
+	layer = ABOVE_HUMAN_LAYER
+
+/obj/structure/flora/shimmering_orb/Initialize()
+	. = ..()
+	set_light(1, 1, 4, 2, "#0066ff")
+	light_inner_range = 5
+	light_max_bright = 1
+	light_outer_range = 3
```
Comment (Naxodile) [inline code/game/objects/structures/flora.dm]: set_light()'s entire purpose is to do this stuff. It doesn't make much sense to set these values twice.
Side note, inner range being greater than outer range is equally strange.

**Comment by ShibberinoShipperino** [comment]
Reworked. Also I have a new git account, etc.

**Pull request closed without merging**